strassengezwitscher/events/models.py

```diff
@@ -11,3 +14,9 @@ class Event(MapObject):
     link_to_event = models.URLField(blank=True, null=True)
```

Comment:
Fände ich okay. Könnte mir auch die Umbenennung in "url" für beide gut vorstellen.